### PR TITLE
issue#43 -  CLI script

### PR DESCRIPTION
**Pull request by gbarat87**
Add a CLI script to apply a set of block from a file in the server.
sudo -u www-data /usr/bin/php admin/tool/blocks_cli_setup.php --filename=block_setup.txt

setup.txt should contain a valid set of blocks

### CODE REVIEW

File: blocks_cli_setup.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<?php
```
Comment (dmitriim) [inline blocks_cli_setup.php:1]: can you please move this file under /cli/blocks_cli_setup.php ?

File: blocks_cli_setup.php [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+list($options, $unrecognized) = cli_get_params($longparams, $shortmappings);
+
+if ($unrecognized) {
+    $unrecognized = implode("\n  ", $unrecognized);
+    cli_error(get_string('cliunknowoption', 'admin', $unrecognized));
+}
+
+if ($options['help'] || empty($options['filepath'])) {
+    echo
+    "CLI script to apply a set of blocks for the plugin tool_blocksmanager from a file.
```
Comment (dmitriim) [inline blocks_cli_setup.php:49]: would be nice to update README and add a note about this script.

File: blocks_cli_setup.php [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+Example:
+\$ sudo -u www-data /usr/bin/php admin/tool/blocks_cli_setup.php --filepath=block_setup.txt
+";
+
+    die;
+}
+
+$file = $options['filepath'];
+
+$data = file_get_contents($CFG->dataroot.'/'.$file);
```
Comment (dmitriim) [inline blocks_cli_setup.php:64]: would be much better to allow the file to be anywhere. Not only under dataroot. Can you please remove `$CFG->dataroot`?

File: blocks_cli_setup.php (hunk 1 above)
Comment (gbarat87) [reply blocks_cli_setup.php]: The file has been moved under cli/

File: blocks_cli_setup.php (hunk 2 above)
Comment (gbarat87) [reply blocks_cli_setup.php]: I have added the part "CLI SCRIPT" In the README file

File: blocks_cli_setup.php (hunk 3 above)
Comment (gbarat87) [reply blocks_cli_setup.php]: $CFG->dataroot have been removed

**Pull request merged**